### DISCUSSION TITLE
Muse 2 & S now supported by Brainflow?

### DISCUSSION THREAD

**Discussion by ErikBjare** [NeuroTechX/EEG-ExPy#121] [General]
Was digging through the docs and found that it seems like Muse devices are supported by Brainflow now? https://brainflow.readthedocs.io/en/stable/SupportedBoards.html#muse-s-bled

Haven't tested it out, but since muse-lsl is pretty dead and unmaintained it might make sense to switch over in the future.

**Comment by JohnGriffiths** [comment]
Yeah it's certainly promising. But currently looks like they only support muse devices via the BLED dongle, that most people don't have. We would need brainflow to support muse with (at least) the native windows Bluetooth hardware before moving off muselsl. Hopefully that's in the works from the brainflow folks.

**Comment by PeterGamma** [comment]
There is an example for BrainFlow Streaming from OpenBCI over BrainFlow to a Rasperry Pi 4, and the OpenBCI GUI can stream to BrainFlow, but I do not know of anyone who has tested BrainFlow with the Muse. Whould be fine if it worked, since there are issues with LSL and Raspberry Pi4, since LSL requires X86 processors. With arm based Raspberry Pi’s it does not work:

https://github.com/alexandrebarachant/muse-lsl/issues/140
